Add Cloud Dedicated docs (#623)
apollographql/docs: src/content/graphos/org/plans.mdx

---
title: Managing your Apollo plan

> subtitle: Understand your Apollo plan, features, and billing

Reviewer: Add SEO description

```suggestion
subtitle: Understand your Apollo plan, features, and billing
description: Learn about Apollo plans available for your organization: Serverless (Free), Serverless, Dedicated, Enterprise
```

> **These are _brief summaries_ of each plan.** For complete plan details, see the [pricing page](https://apollographql.com/pricing).

| Plan                             | Description                                                                                                                                                                                                                                                                                                |
| -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **Serverless (Free)**            | Provides access to serverless [cloud supergraphs](../graphs/#cloud-supergraphs), along with many other GraphOS features. Sets a fixed limit on monthly serverless operations. This plan is free.                                                                                                           |

> | **Serverless**                   | Removes the operation limit included in the **Serverless (Free)** plan, and is billed based on usage _above_ that limit. Also expands metrics data retention.                                                                                                                                              |

Reviewer: ```suggestion
| **Serverless**                   | Removes the operation limit of the **Serverless (Free)** plan, and is billed based on usage _above_ that limit. Also expands metrics data retention.                                                                                                                                              |
```
